File: scripts/Hello_World_Busterz.java
/*
  LANGUAGE: JAVA
  AUTHOR: Deddy T
  GITHUB: https://github.com/Busterz
*/

import java.util.Scanner;

public class Hello_World_Busterz {
    public static void main(String[] args) {
        System.out.println("Hi there, what's your name?");
        
        Scanner in = new Scanner(System.in);
        String user = in.nextLine();
        
        if(user.length() > 0){
            user = user.substring(0, 1).toUpperCase() + user.substring(1, user.length());
        }
        
        System.out.println("Hello " + user + "!");
        System.out.println("Hello World!");
        System.out.println("Welcome to Programming!");
    }
}

File: scripts/Hello_World_Busterz.cs
// LANGUAGE: C#
// AUTHOR: Deddy T
// GITHUB: https://github.com/Busterz

using System;
using System.Collections.Generic;
using System.Text;

namespace Hello_World_Busterz
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hi there, what's your name?");
            string user = Console.ReadLine();
            if(user.Length > 0)
            {
                user = user.Substring(0, 1).ToUpper() + user.Substring(1, user.Length-1);
            }
            Console.WriteLine("Hello " + user + "!");
            Console.WriteLine("Hello World!");
            Console.WriteLine("Welcome to Programming!");
            Console.Read();
        }
    }
}
